### PR TITLE
Stricter conversion checker

### PR DESCRIPTION
**Pull request by isidentical**

### CODE REVIEW

File: Parser/action_helpers.c [hunk 1]
```diff
@@ -1071,6 +1071,19 @@ _PyPegen_check_legacy_stmt(Parser *p, expr_ty name) {
     return 0;
 }
 
+int
+_PyPegen_check_fstring_conversion(Parser *p, Token* symbol, expr_ty conv) {
+    if (symbol->lineno != conv->lineno || symbol->end_col_offset != conv->col_offset) {
```
Comment (pablogsal) [inline Parser/action_helpers.c:1076]: Can you post an example on how to trigger this?

Comment (isidentical) [reply Parser/action_helpers.c:1076]: For the line check:
```py
print(f"""{
        1
        !
        s
}""")
```

For the column check:
```py
print(f"{1! s}")
```

Comment (pablogsal) [reply Parser/action_helpers.c:1076]: Could you add them as tests or are we already covering them?

Comment (isidentical) [reply Parser/action_helpers.c:1076]: We already cover them (this is one of the test failures)

**Review by pablogsal — APPROVED** [review]
To the moon!! 🌜 🚀

**Pull request merged**